lib/src/types/account.ts

```diff
@@ -0,0 +1,58 @@
+import { CurrencyCode } from './common'
+
+export type AccountType = 'BANK' | 'CREDIT'
+export type AccountSubType = 'SAVINGS_ACCOUNT' | 'CHECKINGS_ACCOUNT' | 'CREDIT_CARD'
+
+export type Account = {
+  /** Primary identifier of the account */
+  id: string
+  /** Primary identifier of the Item */
+  itemId: string
+  /** Type of the account */
+  type: AccountType
+  /** Sub type of the account */
+  subtype: AccountSubType
+  /** Account's financial institution number */
+  number: string
+  /** Current balance of the account */
+  balance: number
+  /** Account's name or description */
+  name: string
+  /** Account's name provided by the institution based on the level of client. */
+  marketingName?: string
+  /** Account's owner´s fullname */
+  owner?: string
+  /** Account's owner´s tax number */
+  taxNumber?: string
+  /** ISO Currency code of the account's amounts */
+  currencyCode: CurrencyCode
+  bankData?: BankData
```

Comment:
can we add one-liner comment here as well?

Reply:
Also I've noted that in `pluggy-node` we have all one-line comments, but in `pluggy-js` we have comment blocks at the beginning like so:
```js
/**
 * @typedef ConnectorCredential
 * @type {object}
 * @property {string} label - parameter label that describes it
 * @property {string} name - parameter key name
 * @property {CredentialType} type - type of parameter, used to create the form
 * @property {boolean} mfa - If parameter is used for MFA.
 * @property {string} data - Code for QR image to be resolved (credential type 'image')
 * @property {string} placeholder - Text to use for parameter placeholder in form
 * @property {string} validation - Validation regex to check on the submitted parameter value, before execution
 * @property {string} validationMessage - Validation error message to show to the user
 * @property {boolean} optional - Useful to allow the user to skip/ignoring an unneeded parameter
 * @property {string} assistiveText - Assistive information (supplied by the connector/site) to help the user provide the extra MFA credential
 * @property {CredentialSelectOption[]} options - Available options if credential is of type 'select'
 */
export type ConnectorCredential = {
  label: string
  name: string
  type?: CredentialType
  mfa?: boolean
  data?: string
  placeholder?: string
  validation?: string
  validationMessage?: string
  optional: boolean
  assistiveText?: string
  options?: CredentialSelectOption[]
}
```
Should we agree to have the same format, either one?